Otsu_Thresholding.py

```diff
@@ -0,0 +1,65 @@
+import matplotlib.pyplot as plt
+from skimage.io import imread
+from skimage.filters import threshold_otsu
+
+# -*- coding: utf-8 -*-
+"""
+Created on Wed May  1 11:08:50 2019
+Global Otsu thresholding
+@author: mostly Luis
+"""
+
+def import_image(path, name):
+    # data/image is found in directory and imported as gray scale image
+    raw_image_directory = path + name
+    image = imread(raw_image_directory, as_gray=True)
+    return image
+
+
+def figure(original_image, binary_image, threshold_value):
```

Comment:
A more descriptive name might be appropriate instead of `figure`.

Also, this function does more than one thing:

1. It creates a figure with three subplots
2. It plots the original image
3. It plots a histogram with the threshold
4. It plots the binary image

Each of those functionalities should be handeled by their own respective functions like:

`def plot_image(axes, image, title)`
This can plot general images (original and binary) onto the specified axes object.
`def plot_otsu_hist(axes, image, otsu_threshold)`